medicines/api/Cargo.toml

```diff
@@ -14,10 +14,10 @@ actix-cors = "0.2.0"
 futures = "0.3.1"
 juniper = { git = "https://github.com/graphql-rust/juniper", features = ["async"], rev="211737cee0c6b79ad86246d69ed39e2725fc2110" }
 listenfd = "0.3.3"
-reqwest = { version = "0.10.1", features = ["json"] }
+reqwest = { version = "0.10.4", features = ["json"] }
```

Comment:
are these cargo updates related to enabling sccache?

Reply:
No, I just wanted to trigger a full rebuild and figured a minor version update was probably a good thing to do anyway

Reply:
probably okay if it builds - was worried it was causing unnecessary conflicts but if they're resolved it's no problem